helpme/views/helpme.py

```diff
@@ -35,20 +34,26 @@ def form_valid(self, form):
         user_agent = self.request.user_agent
 
         if user_agent.is_mobile:
-            device = _("Mobile"),
+            device = "Mobile"
         elif user_agent.is_pc:
-            device = _("PC")
+            device = "PC"
         elif user_agent.is_tablet:
-            device = _("Tablet")
+            device = "Tablet"
         else:
-            device = _("Unknown")
+            device = "Unknown"
 
         form.instance.user_meta = {
-            _("Browser"): user_agent.browser.family + " " + user_agent.browser.version_string,
-            _("Operating System"): user_agent.os.family + " " + user_agent.os.version_string,
-            _("Device"): user_agent.device.family,
-            _("Mobile/Tablet/PC"): device,
-            _("IP Address"): self.request.META['REMOTE_ADDR']
+            "Browser": {
```

Comment:
the keys should be all lowercase with optional underscore and should not contain spaces or slashes.